raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -488,10 +504,19 @@ private boolean handleVoteResponse(
         }
     }
 
-    private int randomElectionJitterMs() {
-        if (electionJitterMs == 0)
+    private int randomElectionTimeoutMs() {
+        if (electionTimeoutMs == 0)
             return 0;
-        return random.nextInt(electionJitterMs);
+        // avoiding float calculations here
+        return electionTimeoutMs + random.nextInt(electionTimeoutMs);
+    }
+
+    private int exponentialElectionBackoffMs(int retries) {
+        if (retries == 0)
+            return 0;
+
+        int retryBackOffBaseMs = electionJitterMs / 32;
```

Comment:
5 time retry maximum?

Reply:
That was just for quick benchmarks.

I think a more reasonable way is to set a small enough base milliseconds. I've set it to 100 as static now.